port_ocean/clients/port/mixins/integrations.py

```diff
@@ -50,13 +51,36 @@ async def get_log_attributes(self) -> LogAttributes:
             self._log_attributes = response["logAttributes"]
         return self._log_attributes
 
+    async def _poll_integration_until_default_provisioning_is_complete(
+        self,
+        interval=15,
```

Comment:
15 what?

Reply:
lets not use magic numbers

Reply:
can we do some kind of backoff or eventually mechanism? 15 seconds is huge time to wait for integration to finish set up

Reply:
good idea, will add

Reply:
Fixed!